src/ptvsd/common/attach_script_ptvsd_pid.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root
+# for license information.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os.path
```

Comment:
`import os` is enough - `os.path` is not really a subpackage, and it's documented as such.